### DISCUSSION TITLE
私のアプリはリンクされています。フロントページに表示するにはどうすればいいですか？

### DISCUSSION THREAD

**Discussion by tomtomtong** [pixiv/vroid-sdk-developers#13] [Q&A] [answered]
私のアプリはリンクされています。フロントページに表示するにはどうすればいいですか？
My app is Dan Sing Sing MV Maker
![ps](https://user-images.githubusercontent.com/3156876/136878940-a167587d-430e-4a05-8a8b-45067226ac74.png)

**Comment by Haresaku** [comment, marked as answer]
VRoid Hubトップページへの掲載には [VRoid SDK アプリケーション掲載依頼](https://www.pixiv.net/support?type=47&mode=inquiry&service=vroid-integrate) から依頼をしていただき、サポートチームで審査ののち公開となります。
掲載の依頼はされているようですのでしばらくお待ちください

**Reply by tomtomtong** [reply]
ありがとうございます。引き換えコードを渡す必要がありますか？
このアプリには課金があります。

**Reply by Haresaku** [reply]
そうですね。こちらで確認いたしますので、「お問い合わせ内容」に引き換えコードの内容を記載してご連絡いただけますでしょうか

**Reply by tomtomtong** [reply]
ありがとうございます。アプリの引き換えコードをVRoid お問い合わせフォームにお送りしました。

**Reply by Haresaku** [reply]
Publicにアクセスできるこの場所にURLを記載するのはまずいので消しておきました。
URLはこちらで控えておりますので改めて確認させていただきます

**Reply by tomtomtong** [reply]
ありがとうございました
